recipes/elfx86exts/build.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+set -e
+cargo install --bin elfx86exts --root $PREFIX
```

Comment:
Does this install from the tarball we downloaded or does it download a fresh one from crates?

Reply:
Because it's run from the unpacked tarball directory, it uses the local files. It *does* download the dependencies from crates.io, though. Their precise versions are specified in the `Cargo.lock` file so things should be reproducible, but there is a network access here.

Reply:
Is there any way to suppress this behavior with a flag or similar to `cargo`?

Reply:
@jakirkham, which behaviour do we want to suppress?

Reply:
Note that the dependencies are not packaged as they are only build time dependencies. Final conda package has only one file `bin/elfx86exts` other than the `info` folder.